### DISCUSSION TITLE
URL in documentation points to the old Dockerfile

### DISCUSSION THREAD

**Discussion by Verhoeckx** [gitpod-io/openvscode-server#489] [General] [answered]
The url in README.md that points to the Dockfile, points to an old version of it.

The url is
`https://github.com/gitpod-io/openvscode-releases/blob/eb59ab37e23f8d17532b4af4de37eafaf48037a5/Dockerfile#L64`

And should be (IMHO)
`https://github.com/gitpod-io/openvscode-releases/blob/main/Dockerfile`

**Comment by jeanp413** [comment, marked as answer]
Thanks, updated the link :+1:
